robot_info: 
  max_speed: [0.3, 3.14]
  max_acce: [1.2, 6.28]
  radius: 0.17
  length: 0.34
  width: 0.34
  wheelbase: 0.275
  dynamics: diff
  cone_type: "Rpositive"

# for MPC parameters
receding: 8
iter_num: 2
enable_reverse: false
sample_time: 0.4
process_num: 4
accelerated: True
time_print: False
obstacle_order: True
max_edge_num: 4
max_obs_num: 8
goal_index_threshold: 1

# Tune parameters
iter_threshold: 0.1
slack_gain: 7.0
max_sd: 0.1
min_sd: 0.05
ws: 0.2
wu: 10
ro1: 200
ro2: 1.0

# reference speed
ref_speed: 0.2

# goal generated path
curve_type: 'line'
step_size: 0.2
min_radius: 0.02

# do not use scan
use_scan_obstacle: False
# scan_eps: 0.2
# scan_min_samples: 3

# for frame
target_frame: odom
lidar_frame: laser_frame
base_frame: base_link

marker_lifetime: 0.5            # 标记存活时间 (秒)